atcoder/abc147B_palindrome_philia.go

```diff
@@ -0,0 +1,27 @@
+// andrea denisse
+// https://atcoder.jp/contests/abc147/tasks/abc147_b
+
```

Comment:
add category, for this problem it's "string manipulation"

Reply:
Done.